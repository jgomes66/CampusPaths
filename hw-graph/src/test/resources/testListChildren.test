# Test ListChildren
# Testing to see if listChildren can list the children of given node

# Create a graph
CreateGraph newGraph

# Add all parent and child nodes
AddNode newGraph parent
AddNode newGraph child1
AddNode newGraph child2
AddNode newGraph child3

# Add edges between parent and all child nodes
AddEdge newGraph parent child1 e1
AddEdge newGraph parent child2 e2
AddEdge newGraph parent child3 e3

# List the children of the parent node
ListChildren newGraph parent