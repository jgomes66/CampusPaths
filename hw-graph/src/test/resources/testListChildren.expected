# Test ListChildren
# Testing to see if listChildren can list the children of given node

# Create a graph
create graph newGraph

# Add all parent and child nodes
added node parent to newGraph
added node child1 to newGraph
added node child2 to newGraph
added node child3 to newGraph

# Add edges between parent and all child nodes
added edge e1 from parent to child1 in newGraph
added edge e2 from parent to child2 in newGraph
added edge e3 from parent to child3 in newGraph

# List the children of the parent node
the children of parent in newGraph are: child1(e1) child2(e2) child3(e3)